extensions/gopass/README.md

```diff
@@ -1,5 +1,5 @@
 # A gopass extension for Raycast
 
 To use this extension, you have to have [gopass](https://www.gopass.pw/)
-installed into `/usr/bin` or `/usr/local/bin` (default path when using
-[Homebrews](https://brew.sh/)).
+installed into `/usr/bin`, `/usr/local/bin` (default path when using
+[Homebrews (macOS - Intel)](https://brew.sh/)) or `/opt/homebrew/bin` [Homebrews (macOS - Apple Silicon)](https://docs.brew.sh/FAQ#why-is-the-default-installation-prefix-opthomebrew-on-apple-silicon))
```

Comment:
I made a typo and called it "Homebrews"! Could you please fix them both?

Reply:
Thanks! I fixed it.